app/org/sagebionetworks/bridge/play/controllers/AuthenticationController.java

```diff
@@ -102,7 +87,40 @@ public Result resetPassword() throws Exception {
         authenticationService.resetPassword(passwordReset);
         return okResult("Password has been changed.");
     }
-    
+
+    private Result signInWithRetry(final int retryCounter) throws Exception {
+
+        UserSession session = getSessionIfItExists();
+        if (session != null) {
+            setSessionToken(session.getSessionToken());
+            return okResult(new UserSessionInfo(session));
+        }
+
+        final JsonNode json = requestToJSON(request());
+        final SignIn signIn = parseJson(request(), SignIn.class);
+        final Study study = getStudyOrThrowException(json);
+        try {
+            Logger.info("User " + signIn.getUsername() + " signing in for study " + study.getIdentifier() + ".");
```

Comment:
Do we need to log this for *every* sign in? Or just for the ones that throw?

Also, I'm concerned that the user name could be identifying info. Are we able to log these in the logs? Would it be safer to log a hash of the username, so we can see if/when they repeat without having to expose the actual user name?